vtkext/private/module/vtkF3DRenderer.cxx

```diff
@@ -2769,6 +2781,79 @@ bool vtkF3DRenderer::ConfigureVolumeForColoring(vtkSmartVolumeMapper* mapper, vt
   return true;
 }
 
+//----------------------------------------------------------------------------
+void vtkF3DRenderer::EnableJitter(bool enable)
+{
+  vtkActorCollection* actors = this->GetActors();
+  actors->InitTraversal();
+  vtkActor* actor;
+
+  float jitter[2];
+  if (enable)
+  {
+    jitter[0] = this->ConfigureHaltonSequence(0);
+    jitter[1] = this->ConfigureHaltonSequence(1);
+
+    vtkRenderWindow* renderWindow = this->GetRenderWindow();
+    int width = renderWindow->GetSize()[0];
+    int height = renderWindow->GetSize()[1];
+
+    jitter[0] = ((jitter[0] - 0.5f) / width) * 2.0f;
+    jitter[1] = ((jitter[1] - 0.5f) / height) * 2.0f;
+  }
+  else
+  {
+    jitter[0] = 0.0f;
+    jitter[1] = 0.0f;
+  }
+
+  while ((actor = actors->GetNextActor()))
+  {
+    const vtkPolyDataMapper* mapper = vtkPolyDataMapper::SafeDownCast(actor->GetMapper());
+    if (!mapper)
+    {
+      continue;
+    }
+
+    vtkShaderProperty* shaderProp = actor->GetShaderProperty();
+    vtkUniforms* uniforms = shaderProp->GetVertexCustomUniforms();
+    uniforms->SetUniform2f("jitter", jitter);
+  }
+
+  if (enable)
+  {
+    vtkInformation* information = this->GetInformation();
+    information->Remove(vtkF3DRenderPass::RENDER_UI_ONLY());
+  }
+}
+
+//----------------------------------------------------------------------------
+float vtkF3DRenderer::ConfigureHaltonSequence(int direction)
+{
+  int base = (direction == 0) ? 2 : 3;
+  int& n = (direction == 0) ? TaaNx : TaaNy;
+  int& d = (direction == 0) ? TaaDx : TaaDy;
```

Comment:
Can you group `TaaNx` and `TaaNy` to `TaaN[2]` and similarly for `TaaD`?  
That way would can avoid branching and the code will be cleaner imo:

```suggestion
  assert(direction == 0 || direction == 1);
  int base = 2 + direction;
  int& n = TaaN[direction];
  int& d = TaaD[direction];
```